tests/data/server-content/spring.io/tools/sts/all

```diff
@@ -0,0 +1,352 @@
+<p class="file-type">zip</p>
+<span class="file-size">430MB</span>
+</a>
+</div>
+</div>
+</div>
+<div class="item--half-column">
+<p class="item--body-title">WIN, 64BIT</p>
+<div>
+<div class="item--body--version">
+<a class="tool-download-link" target="_blank" href="http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-win32-x86_64.zip" onclick="_gaq.push([&#39;_trackEvent&#39;,&#39;download STS 3.7.2.RELEASE windows&#39;,&#39;http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-win32-x86_64.zip&#39;])">
+<p class="file-type">zip</p>
+<span class="file-size">430MB</span>
+</a>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div class="platform js-item-dropdown-widget--wrapper">
+<div class="mac-icon platform--icon"></div>
+<h3>Mac</h3>
+<div class="item-dropdown-widget js-item-dropdown--wrapper">
+<div class="item-dropdown--link js-item--open-dropdown">
+<p class="item-dropdown--title tools-version-dropdown--title">Based on Eclipse 4.5.1</p>
+<i class="item-dropdown--icon icon-reorder place-right"></i>
+</div>
+<div class="item--dropdown no-item--header platform-version--dropdown">
+<div class="item--body">
+<div class="item--half-column">
+<p class="item--body-title">COCOA, 64BIT</p>
+<div>
+<div class="item--body--version">
+<a class="tool-download-link" target="_blank" href="http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-macosx-cocoa-x86_64.tar.gz" onclick="_gaq.push([&#39;_trackEvent&#39;,&#39;download STS 3.7.2.RELEASE mac&#39;,&#39;http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-macosx-cocoa-x86_64.tar.gz&#39;])">
+<p class="file-type">tar.gz</p>
+<span class="file-size">428MB</span>
+</a>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div class="platform js-item-dropdown-widget--wrapper">
+<div class="linux-icon platform--icon"></div>
+<h3>Linux</h3>
+<div class="item-dropdown-widget js-item-dropdown--wrapper">
+<div class="item-dropdown--link js-item--open-dropdown">
+<p class="item-dropdown--title tools-version-dropdown--title">Based on Eclipse 4.5.1</p>
+<i class="item-dropdown--icon icon-reorder place-right"></i>
+</div>
+<div class="item--dropdown no-item--header platform-version--dropdown">
+<div class="item--body">
+<div class="item--half-column">
+<p class="item--body-title">GTK, 32BIT</p>
+<div>
+<div class="item--body--version">
+<a class="tool-download-link" target="_blank" href="http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-linux-gtk.tar.gz" onclick="_gaq.push([&#39;_trackEvent&#39;,&#39;download STS 3.7.2.RELEASE linux&#39;,&#39;http://dist.springsource.com/release/STS/3.7.2.RELEASE/dist/e4.5/spring-tool-suite-3.7.2.RELEASE-e4.5.1-linux-gtk.tar.gz&#39;])">
```

Comment:
I thought that as STS is eclipse-based i should test it similarly, the java-fake script includes the arch of the machine, so i guess they should be different tarballs